docs/reference/repositories-metering-api/apis/repositories-meterings-body.asciidoc

```diff
@@ -0,0 +1,135 @@
+tag::repositories-metering-body[]
+`<node_id>`::
+(Optional, string) Comma-separated list of node IDs or names used to limit
+returned information.
+`_nodes`::
+(object)
+Contains statistics about the number of nodes selected by the request.
++
+.Properties of `_nodes`
+[%collapsible%open]
+====
+`total`::
+(integer)
+Total number of nodes selected by the request.
+
+`successful`::
+(integer)
+Number of nodes that responded successfully to the request.
+
+`failed`::
+(integer)
+Number of nodes that rejected the request or failed to respond. If this value
+is not `0`, a reason for the rejection or failure is included in the response.
+====
+
+`cluster_name`::
+(string)
+Name of the cluster. Based on the <<cluster.name>> setting.
+
+`nodes`::
+(object)
+Contains repositories metrics for the nodes selected by the request.
++
+.Properties of `nodes`
+[%collapsible%open]
+====
+`<node_id>`::
+(array)
+An array of repository metrics for the node.
++
+.Properties of objects in `node_id`
+[%collapsible%open]
+=====
+`repository_name`::
+(string) 
+Repository name.
+
+`repository_type`::
+(string) 
+Repository type.
+
+`repository_location`::
+(string) 
+Represents an unique location within the repository, 
+i.e. a `bucket` + `base_path`.
```

Comment:
This should probably be defined more clearly for all supported repository types? Also will there be a need for a client of this API (i.e. Cloud) to deconstruct the location back into bucket and base_path? If so, separate (repository-type specific) fields might be better, similar to how the "request_counts" object works?